submissions/Frankenstei/Week 3/about week.md

```diff
@@ -0,0 +1 @@
+Week  3 solutions
```

Comment:
Please transfer your week 3 task to Google docs and share directly with your mentors